graphbolt/src/csc_sampling_graph.cc

```diff
@@ -97,6 +97,48 @@ void CSCSamplingGraph::Save(torch::serialize::OutputArchive& archive) const {
   }
 }
 
+void CSCSamplingGraph::SetState(
+    torch::Dict<std::string, torch::Dict<std::string, torch::Tensor>>& state) {
+  auto&& regular_tensors = state.at("regular_tensors");
```

Comment:
worth to add a comment to explain why we use && here.

Reply:
What about just 
```suggestion
  auto&& tensors = state.at("tensors");
```

Reply:
I think `const auto &` is enough. No need to use `&&`.